SignItCoreContent.js

```diff
@@ -62,9 +51,42 @@ var SignItCoreContent = function (locale,mapi18n) {
     this.$definitionPanelSpinner = this.$container.find(".signit-loading");
     this.$definitionPanelError = this.$container.find(".signit-error");
 
-    // this.contributeButton.on( 'click', function () {
-    //	// TODO: Do something
-    // }.bind( this ) );
+    SignItCoreContent.prototype.init = async function () {
+      try {
+        var banana = {i18n: async (msg,...arg) => {
+          return await chrome.runtime.sendMessage({
+            command:'bananai18n',arg : [msg,arg]
+          })
+        }
+      };
+        const translations = await Promise.all([
+          banana.i18n("si-panel-videos-title"),
+          banana.i18n("si-panel-videos-empty"),
+          banana.i18n("si-panel-definitions-title"),
+          banana.i18n("si-panel-videos-contribute-label"),
+          banana.i18n("si-panel-definitions-wikt-iso"),
+          banana.i18n("si-panel-definitions-wikt-pointer"),
+          banana.i18n("si-panel-definitions-empty") // May need a different key for error message
+        ]);
+        const [videosPanelNoVideoTitle, videosPanelNoVideoEmpty, definitionsPanelTitle, contributeButtonLabel, wiktIso, wiktPointer, definitionsEmpty] = translations;
```

Comment:
I don't see raison for the negation `No`, nor why `videos` appear at begging while the general convention so far is from wider to specific such as `SI > Panel > Videos|Definitions > [message]`. 

Variable names suggested : 
- `videosPanelNoVideoTitle` -> `panelVideosTitle`
- `videosPanelNoVideoEmpty` --> `panelVideosEmpty`
- `definitionsPanelTitle` --> `panelDefinitionsTitle`
-  `contributeButtonLabel` --> `panelVideosContributeLabel`
- `wiktIso`
- `wiktPointer`
- `definitionsEmpty` - > `panelDefinitionsEmpty`

I believe the prefix `panel` with the camelCase ensure a WebExt-wide uniqueness.

Reply:
Guess I'll change it then